arm/entrypoint.sh

```diff
@@ -2,60 +2,77 @@
 
 set -e
 
-# Create Resource group if it does not exists
-
-RESOURCE_GROUP_EXISTS=$(az group exists -n "$AZURE_RESOURCE_GROUP")
-if [[ $RESOURCE_GROUP_EXISTS == "false" ]]
+if [[ -z "$AZURE_RESOURCE_GROUP" ]]
 then
-  if [[ -z "$RESOURCE_GROUP_LOCATION" ]]
+  echo "AZURE_RESOURCE_GROUP is not set." >&2
+  exit 1
+else
+  if [[ -z "$AZURE_RG_COMMAND" ]] || [[ ${AZURE_RG_COMMAND,,} == 'create' ]]
```

Comment:
what does ,, indicate?

Reply:
that evaluates the AZURE_RG_COMMAND to lower case. i added this for case insensitive command